src/main/scala/com/nec/ve/colvector/VeColVector.scala

```diff
@@ -145,17 +143,17 @@ final case class VeColVector(underlying: GenericColVector[Long]) {
       if (numItems > 0) {
         val dataSize = numItems * 8
         float8Vector.setValueCount(numItems)
-        val vhTarget = ByteBuffer.allocateDirect(dataSize)
-        val validityTarget = ByteBuffer.allocateDirect(numItems)
+        val vhTarget = new BytePointer(ByteBuffer.allocateDirect(dataSize))
+        val validityTarget = new BytePointer(ByteBuffer.allocateDirect(numItems))
```

Comment:
We don't need to call those allocateDirect() here or anywhere else. Use BytePointer to allocate memory.

Reply:
Right, missed those, will remove.